plugins/auto-sizes/readme.txt

```diff
@@ -7,12 +7,11 @@ License:      GPLv2 or later
 License URI:  https://www.gnu.org/licenses/gpl-2.0.html
 Tags:         performance, images, auto-sizes
 
-Instructs browsers to automatically choose the right size for images.
+Improvements for WordPress' responsive image functionality.
 
 == Description ==
 
-This plugin implements the HTML spec for adding `sizes="auto"` to lazy-loaded images.
-For background, see: https://github.com/whatwg/html/issues/4654.
+This plugin implements experimental enhancements for the responsive images functionality in WordPress. Currently, this includes improvements to the accuracy of the `sizes` attribute by using available layout information in the theme and by implementing the new HTML spec for adding `sizes="auto"` to lazy-loaded images (See: https://github.com/whatwg/html/issues/4654).
 
```

Comment:
```suggestion
There are currently **no settings** and no user interface for this plugin since it is designed to work without any configuration.

```

I suggested this in #1322 as well.